### PR TITLE
Cleanup existing README

### PR DESCRIPTION
**Pull request by apenney**
Tighten up the wording, reorganize the features list a little, and
move the giant video section to below the install section so people
know how to quickstart as soon as possible.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1611)

### CODE REVIEW

**Comment by apenney** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -12,34 +12,20 @@ The easiest way to get a production grade Kubernetes cluster up and running.
...
-Check out our [tutorial](/docs/aws.md) for launching a Kubernetes cluster hosted
-on AWS.
-
-<p align="center">
-  <a href="https://asciinema.org/a/97298">
-  <img src="https://asciinema.org/a/97298.png" width="885"></image>
-  </a>
-</p>
-
+`kops` lets you deploy production-grade, with highly available masters,
```
Comment (kris-nova) [inline README.md]: This wording seems a bit off.. Suggest:

> `kops` lets you deploy production-grade, highly available Kubernetes clusters from the command line.

File: README.md [hunk 2]
```diff
@@ -12,34 +12,20 @@ The easiest way to get a production grade Kubernetes cluster up and running.
...
-* Highly Available (HA) Kubernetes Masters Setup
-* Uses a state-sync model for **dry-run** and automatic **idempotency**
-* Custom support for `kubectl` [add-ons](/docs/addons.md)
-* Kops can generate [Terraform configuration](/docs/terraform.md)
-* Based on a simple meta-model defined in a directory tree
-* Command line [autocomplete](/docs/cli/kops_completion.md)
-* Community support
-* [Upgrade from kube-up](/docs/upgrade_from_kubeup.md)
-
+* Automate the provisioning of Kubernetes clusters in ([AWS](/docs/aws.md))
```
Comment (chrislovecnm) [inline README.md]: Should we do singular here?

File: README.md [hunk 3]
```diff
@@ -81,6 +67,17 @@ See [building notes](/docs/build.md) for more information.
 
 At this time, Windows is not a supported platform.
 
+## Launching a Kubernetes cluster hosted on AWS
+
+Check out our [tutorial](/docs/aws.md) for launching a Kubernetes cluster hosted
+on AWS.
+
+<p align="center">
```
Comment (chrislovecnm) [inline README.md]: I am torn... the gif is flashy. Is this moving it below the top of the fold?

File: README.md [hunk 4]
```diff
@@ -112,13 +109,8 @@ A great place to get involved or ask questions is [#sig-cluster-lifecycle](https
  - Look at our [other interesting modes](/docs/commands.md#other-interesting-modes)
  - Full command line interface [documentation](/docs/cli/kops.md)
 
-## History
-
```
Comment (chrislovecnm) [inline README.md]: Probably should link the releases tab here. What does the core repo do?

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

File: README.md (hunk 3 above)
Comment (apenney) [inline README.md]: I felt the same way, it's attention drawing and I love that, maybe I'll change my mind on this.  I was thinking "I want to install this! > I want to see it" but maybe having the focus be on "Why should I install this anyway?" makes sense.

I'll handle the rest of the comments today!

File: README.md (hunk 2 above)
Comment (apenney) [inline README.md]: Hmm, I think one of the benefits is you can easily control multiple clusters, so singular would seem a bit weird.  We're going to have qa/staging/prod clusters and we can't be too unique!

**Comment by apenney** [comment]
Alright, take two, bringing the demo front and center!

File: README.md (hunk 4 above)
Comment (kris-nova) [inline README.md]: I am fine with cleaning this up for now.. we can (and should) address the changelog during our next drop (1.5.0)

Comment (kris-nova) [inline README.md]: EG: Another PR...

**Comment by kris-nova** [comment]
@apenney thanks again for the PR! Waiting to hear @chrislovecnm's response on the changelog.. but LGTM.. thanks again

**Pull request merged**